words/words.md

```diff
@@ -695,25 +695,25 @@ Lass uns abschließend zurückkehren zu...
     </div>
 </div>
 
-Das Flugzeug ist gesunken. Zusammengedrängt hocken wir in den Rettungsinseln. Es ist Zeit, Land zu suchen.[^dry_land]
+Das Flugzeug ist versunken. Wir hocken zusammengedrängt in den Rettungsbooten. Es ist Zeit, Festland zu suchen.[^dry_land]
 
 [^dry_land]: Festland-Metapher [von Marc Lipsitch und Yonatan Grad, auf STAT News](https://www.statnews.com/2020/04/01/navigating-covid-19-pandemic/)
 
-Teams aus Epidemiologinnen und Menschen in der Politik ([links](https://www.americanprogress.org/issues/healthcare/news/2020/04/03/482613/national-state-plan-end-coronavirus-crisis/), [rechts](https://www.aei.org/research-products/report/national-coronavirus-response-a-road-map-to-reopening/ ), und [überparteilich](https://ethics.harvard.edu/covid-roadmap)) haben einen Konsens darüber erzielt, wie wir COVID-19 schlagen können und gleichzeitig unser Leben *und* unsere Freiheiten schützen.
+Teams aus Epidemiologinnen und Epidemiologen und Menschen in der Politik ([links](https://www.americanprogress.org/issues/healthcare/news/2020/04/03/482613/national-state-plan-end-coronavirus-crisis/), [rechts](https://www.aei.org/research-products/report/national-coronavirus-response-a-road-map-to-reopening/ ), und [überparteilich](https://ethics.harvard.edu/covid-roadmap)) haben einen Konsens darüber erzielt, wie wir COVID-19 schlagen können und gleichzeitig unser Leben *und* unsere Freiheiten schützen.
```

Comment:
In den USA ist das politische Gefüge in der Politik recht anders als bei uns. Kann jemand was dazu sagen ob wir hier links/rechts einfach so direkt übersetzt übernehmen können? Die etablierten Parteien in den USA sind aus unserer Sicht alle eher konservativ. Die verlinkten Artikel könnte ich jetzt politisch nicht so einfach einordnen (bzw. müsste sie vorher lesen). Evtl. sehr wortreich: "im linken Spektrum der US-Politik" / "im rechten Spektrum der US-Politik"?

Reply:
DIe Organisationen, die hinter den verlinkten Artikeln stehen, lassen sich meiner kurzen Recherche nach sehr eindeutig als "progressiv", "konservativ" und "überparteilich" bezeichnen.